doc/zsh-completion.zsh

```diff
@@ -705,7 +705,7 @@ function _restic_self-update {
 
 function _restic_snapshots {
   _arguments \
-    '(-c --compact)'{-c,--compact}'[use compact format]' \
+    '(-c --compact)'{-c,--compact}'[compact printed output]' \
```

Comment:
Please remove this change, as the completion files are generated automatically by the release build scripts.

Reply:
If I'm correct, you made the same comment 2x: https://github.com/restic/restic/pull/2955#discussion_r498499011

Reply:
May be, I wrote a comment where it was applicable to do so. The changes to this file still need to be removed from the PR.